Index.html

```diff
@@ -35,19 +35,38 @@
 
     <main>
       <div class="carousel">
-        <div class="carousel-images">
+        <div class="carousel-item">
+          <div  class="carousel-text">
+            <p class="carousel-text__title">made with love</p>
```

Comment:
You used carousel-text__**title** css class on it, but didn't use a `<h2>` or simmilar tag on it, but used a `<p>`?